common/lib/xmodule/xmodule/tabs.py

```diff
@@ -375,6 +376,10 @@ def __init__(self, tab_dict=None):
         )
 
     def can_display(self, course, settings, is_user_authenticated, is_user_staff, is_user_enrolled):
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
+            from ccx.overrides import get_current_ccx
+            if get_current_ccx():
```

Comment:
Seems like there are a few places that examine the feature flag, and then get_current_ccx.  It might help to have a single function that combines the two.